src/components/ResultsBody/PaginatedList/usePaginatedList.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import { api } from "../../../shared/api.js";
+
+export const useAllVenues = () => {
```

Comment:
The hook name is misleading, because the file is `usePaginetedList.js`

Reply:
done